src/integrationTest/java/org/openstreetmap/atlas/mutator/AtlasMutatorCountryvoreIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package org.openstreetmap.atlas.mutator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.generator.AtlasGeneratorParameters;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+
+/**
+ * @author matthieun
+ */
+public class AtlasMutatorCountryvoreIntegrationTest extends AbstractAtlasMutatorIntegrationTest
```

Comment:
Pointing out `Countryvore` which I assume is a typo. If not, then I would add a comment explaining what it means!

Reply:
Haha! ok!